sopel/tools/__init__.py

```diff
@@ -485,6 +485,7 @@ def is_nick(self):
         :return: ``True`` if this :py:class:`Identifier` is a nickname;
                  ``False`` if it appears to be a channel
         """
+        # TODO: Handle CHANTYPES from ISUPPORT numeric (005)
```

Comment:
True, though might be worth a note in the docstring to not surprise people?

Reply:
I'm trying to avoid adding more stuff that we'll have to remember to remove later, lol. To my knowledge, not one person has ever been surprised that something non-standard like `^$channel` was treated as `is_nick()` by the `Identifier` type. If they were, they never told anyone involved with the project. 🤷‍♂️

Reply:
Fair, but I figured if that's ever implemented it'll touch that function, so not too far-flung.